pegasus/sites.v3/code.org/public/congrats.haml

```diff
@@ -54,7 +54,7 @@ max_age: 60
         =TextRender.safe_markdown company['additional_actions_copy_t']
       -else
         =view :company_profile_additional_actions
-      = I18n.t :congratulations_volunteer
+      = CGI.escapeHTML(I18n.t(:congratulations_volunteer))
```

Comment:
Shouldn't this (and all the others) be `CGI.unescapeHTML`?

Reply:
d'oh